submarine-sdk/pysubmarine/submarine/artifacts/repository.py

```diff
@@ -19,45 +19,36 @@
 
 
 class Repository:
-    def __init__(self, experiment_id: str):
+    def __init__(self):
         self.client = boto3.client(
             "s3",
             aws_access_key_id=os.environ.get("AWS_ACCESS_KEY_ID"),
             aws_secret_access_key=os.environ.get("AWS_SECRET_ACCESS_KEY"),
             endpoint_url=os.environ.get("MLFLOW_S3_ENDPOINT_URL"),
         )
-        self.dest_path = experiment_id
         self.bucket = "submarine"
 
     def _upload_file(self, local_file: str, bucket: str, key: str) -> None:
         self.client.upload_file(Filename=local_file, Bucket=bucket, Key=key)
 
-    def _list_artifact_subfolder(self, artifact_path: str):
+    def list_artifact_subfolder(self, dest_path):
         response = self.client.list_objects(
             Bucket=self.bucket,
-            Prefix=os.path.join(self.dest_path, artifact_path) + "/",
+            Prefix=f"{dest_path}/",
             Delimiter="/",
         )
         return response.get("CommonPrefixes")
 
-    def log_artifact(self, local_file: str, artifact_path: str) -> None:
-        dest_path = self.dest_path
-        dest_path = os.path.join(dest_path, artifact_path)
+    def log_artifact(self, dest_path: str, local_file: str) -> None:
+        dest_path = os.path.join(dest_path)
```

Comment:
we didn't join any other path here, could we remove this line?

Reply:
Sure